star-sys/STAR/source/MmapArray.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MMAPARRAY_DEF
+#define MMAPARRAY_DEF
```

Comment:
STAR seems to be rather inconsistent about the form of the header guard define.  I would suggest just going with the [Google style guide](https://google.github.io/styleguide/cppguide.html#The__define_Guard) on this.
```suggestion
#ifndef MMAPARRAY_H_
#define MMAPARRAY_H_
```